core/src/main/java/com/datastax/oss/driver/internal/core/context/StartupOptionsBuilder.java

```diff
@@ -142,4 +152,28 @@ protected String getDriverName() {
   protected String getDriverVersion() {
     return Session.OSS_DRIVER_COORDINATES.getVersion().toString();
   }
+
+  private Optional<String> driverBaggage() {
+    Joiner joiner = Joiner.on(": ");
+    Map<String, Optional<String>> lbpToBag =
+        Maps.transformValues(context.getLoadBalancingPolicies(), this::getDriverBaggage);
+    return Optional.of(
+        "{"
+            + lbpToBag.entrySet().stream()
+                .filter(e -> e.getValue().isPresent())
+                .map(
+                    entry ->
+                        joiner.join(Strings.doubleQuote(entry.getKey()), entry.getValue().get()))
+                .collect(Collectors.joining(", "))
+            + "}");
```

Comment:
Why not depend on Jackson to do the serialization?

I was thinking you could add a DriverBaggage class with `String toJson()` and `static DriverBaggage put(String key, DriverBaggage value)`, and in StartupOptionsBuilder here add all the load balancing policies' baggage. Then we wouldn't have the implicit "String that's actually JSON" type.